lua/lspconfig/server_configurations/cairo_ls.lua

```diff
@@ -1,6 +1,6 @@
 local util = require 'lspconfig.util'
 
-local bin_name = 'cairo-language-server'
+local bin_name = 'scarb-cairo-language-server'
 local cmd = { bin_name, '/C', '--node-ipc' }
```

Comment:
these can be both removed just inline cmd into return cmd field.

Reply:
Alright. Will do